src/ws_server.cu

```diff
@@ -166,19 +172,26 @@ class AprilTagHandler : public seasocks::WebSocket::Handler {
     return true;
   }
 
-  // TODO: Implement a faster version of this method!
+/*
```

Comment:
no point keeping this around, dead code rots!  Delete pls

Reply:
sweet fixed